math/mathcore/src/TRandom.cxx

```diff
@@ -15,40 +15,40 @@
-generator</a>, that it should not be used because its period is only 2**31, i.e. approximatly 2 billion events, that can be generated in just few seconds. 
-
-To generate random numbers, one should use the derived class, which  are : 
-- TRandom3: it is based on the "Mersenne Twister generator", 
-it is fast and a very long period of about \f$10^{6000}\f$. However it fails some of the most stringent tests of the 
-<a href="http://simul.iro.umontreal.ca/testu01/tu01.html">TestU01 suite</a>. 
-In addition this generator provide only numbers with 32 random bits, which might be not sufficient for some application based on double or extended precision. 
-This generator is however used in ROOT used to instantiate the global pointer to the ROOT generator, *gRandom*. 
-- ::TRandomMixMax: Generator based on the family of the MIXMAX matrix generators (see the <a href="https://mixmax.hepforge.org">MIXMAX HEPFORGE Web page</a> and the 
-  the documentation of the class ROOT::Math::MixMaxEngine for more information), that are base on the Asanov dynamical C systems. 
-This generator has a state of N=240 64 bit integers, proof random properties, it provides 61 random bits and it has a very large period (\f$10^{4839}\f$). 
-Furthermore, it provides the capability to be seeded with the guarantee that, for each given different seed, a different sequence of random numbers will be generated. 
-The only drawback is that the seeding time is time consuming, of the order of 0.1 ms, while the time to generate a number is few ns (more than 10000 faster).  
-- ::TRandomMixMax17: Another MixMax generator, but with a smaller state, N=17, and this results in a smaller entropy than the generator with N=240. However, it has the same seeding capabilities, with a much faster seeding time (about 200 times less than TRandomMixMax240 and comparable to TRandom3). 
-- ::TRandomMixMax256 : A variant of the MIXMAX generators, based on a state of N=256, and described in the 
+generator</a>, that it should not be used because its period is only 2**31, i.e. approximatly 2 billion events, that can be generated in just few seconds.
+
+To generate random numbers, one should use the derived class, which  are :
+- TRandom3: it is based on the "Mersenne Twister generator",
+it is fast and a very long period of about \f$10^{6000}\f$. However it fails some of the most stringent tests of the
+<a href="http://simul.iro.umontreal.ca/testu01/tu01.html">TestU01 suite</a>.
+In addition this generator provide only numbers with 32 random bits, which might be not sufficient for some application based on double or extended precision.
+This generator is however used in ROOT used to instantiate the global pointer to the ROOT generator, *gRandom*.
+- ::TRandomMixMax: Generator based on the family of the MIXMAX matrix generators (see the <a href="https://mixmax.hepforge.org">MIXMAX HEPFORGE Web page</a> and the
```

Comment:
What do the `::` do? I don't think they change anything for doxygen.

Reply:
The `::` is needed for creating the Doxygen links for the typedef's, see https://www.doxygen.nl/manual/autolink.html
As you see in the current generated doc, those links work, while the one for `TRandomMT64` it does not. I will fix this as also the links to the functions below.

Thnak you for the useful comments and fixes !